docs/installation/index.md

```diff
@@ -29,6 +29,14 @@ pip install helics
 pip install helics-apps
 ```
 
+OR
+
+Use Spack (requires Spack develop branch or versions released after v0.14.1) on HPC systems (Linux/macOS) to install C/C++ HELICS components and apps:
+
+```bash
+spack install helics
+```
```

Comment:
I believe in the spack configuration you had some variants that would work as well,  You might want to explicitly state for example how to install with spack including MPI or something to that effect.

Reply:
or at least point to where you get more details in the next file

Reply:
The read the docs page on package managers talks about it. Maybe the top of quick start should say any of these package managers may support other options, see that page for details and the manuals for your respective package managers?